docs/topics/wasm/wasm-overview.md

```diff
@@ -1,17 +1,17 @@
 [//]: # (title: Kotlin Wasm)
 
-> Kotlin Wasm is [Alpha](components-stability.md).
-> It may be changed at any time. You can use it in scenarios before production. We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+> Kotlin Wasm is in [Alpha](components-stability.md).
```

Comment:
You don't use "in" in other texts for Wasm. So I would remove it for consistency 😄